scripts/cypress/integration/common/5a_user_role.js

```diff
@@ -177,48 +177,64 @@ export const genTest = (apiType, dbType) => {
       it(`[${roles[roleType].name}] Left navigation menu, New User add`, () => {
         // project configuration settings
         //
-        _advSettings(roleType, "userRole");
+        if (roleType !== "owner") {
+          _advSettings(roleType, "userRole");
+        }
       });
 
       it(`[${roles[roleType].name}] Access control`, () => {
         // Access control validation
         //
-        _accessControl(roleType, "userRole");
+        if (roleType !== "owner") {
+          _accessControl(roleType, "userRole");
+        }
       });
 
       it(`[${roles[roleType].name}] Schema: create table, add/modify/delete column`, () => {
         // Schema related validations
         //  - Add/delete table
         //  - Add/Update/delete column
         //
-        _editSchema(roleType, "userRole");
+        if (roleType !== "owner") {
+          _editSchema(roleType, "userRole");
+        }
       });
 
       it(`[${roles[roleType].name}] Data: add/modify/delete row, update cell contents`, () => {
         // Table data related validations
         //  - Add/delete/modify row
         //
-        _editData(roleType, "userRole");
+        if (roleType !== "owner") {
+          _editData(roleType, "userRole");
+        }
       });
 
       it(`[${roles[roleType].name}] Comments: view/add`, () => {
         // read &/ update comment
         //      Viewer: only allowed to read
         //      Everyone else: read &/ update
         //
-        _editComment(roleType, "userRole");
+        if (roleType !== "owner") {
+          _editComment(roleType, "userRole");
+        }
       });
 
       it(`[${roles[roleType].name}] Right navigation menu, share view`, () => {
         // right navigation menu bar
         //      Editor/Viewer/Commenter : can only view 'existing' views
         //      Rest: can create/edit
-        _viewMenu(roleType, "userRole");
+        if (roleType !== "owner") {
+          _viewMenu(roleType, "userRole");
+        }
       });
 
       it(`[${roles[roleType].name}] Download files`, () => {
         // to be fixed
```

Comment:
what is to be fixed? or already handled? better elaborate more with a TODO tag.

Reply:
@dstala - Can you add more details with comment?

Reply:
skipped tests for role `owner` as we do all sort of tests with that credentials already. will update details in comment